Library/OcCpuLib/OcCpuLib.c

```diff
@@ -913,8 +915,20 @@ ScanAmdProcessor (
     Cpu->CurBusRatio = Cpu->MaxBusRatio;
     Cpu->MinBusRatio = Cpu->MaxBusRatio;
 
-    Cpu->FSBFrequency = DivU64x32 (Cpu->CPUFrequency, Cpu->MaxBusRatio);
-  }
+    if (Cpu->CPUFrequency == 0) {
```

Comment:
This if makes no sense to me, as it is always true. Perhaps it is not meant to be there.